Homework11/Homework11/Catalog.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.Json;
+
+namespace Homework11
+{
+    public class Catalog
+    {
+        public List<Product> Products { get; set; }
+
+        public Catalog()
+        {
+            Products = new List<Product>();
+        }
+
+        public void AddProduct(List<Product> products)
+        {
+            foreach (var product in products)
+            {
+                Products.Add(product);
+            }
```

Comment:
fyi - alternativne by take slo pouzit `Products.AddRange(products);`
```suggestion
            Products.AddRange(products);
```